src/golang/lib/job/k8s.go

```diff
@@ -152,6 +173,12 @@ func containerStatusFromPod(pod *corev1.Pod, name string) (*corev1.ContainerStat
 }
 
 func (j *k8sJobManager) Poll(ctx context.Context, name string) (shared.ExecutionStatus, JobError) {
+	if j.k8sClient == nil {
```

Comment:
Do we expect this to ever happen? I thought we always expect to call Launch() before Poll().

Reply:
The other order may happen as we discussed in person.